app.js

```diff
@@ -137,6 +137,13 @@ lookAndFeel.configure(app, {
       isArray(value) {
         return Array.isArray(value);
       },
+      inactivityTimeout: {
+        title: `${content.inactivityTimeout.title}`,
```

Comment:
These don't need to be template literals `${content.inactivityTimeout.title}` can just be `content.inactivityTimeout.title`